src/vsr.zig

```diff
@@ -542,17 +712,20 @@ test "exponential_backoff_with_jitter" {
 /// * A replica's IP address may be changed without reconfiguration.
 /// This does require that the user specify the same order to all replicas.
 /// The caller owns the memory of the returned slice of addresses.
-/// TODO Unit tests.
 /// TODO Integrate into `src/cli.zig`.
 pub fn parse_addresses(allocator: *std.mem.Allocator, raw: []const u8) ![]std.net.Address {
-    var addresses = try allocator.alloc(std.net.Address, config.replicas_max);
+    return parse_addresses_limit(allocator, raw, config.replicas_max);
+}
+
+fn parse_addresses_limit(allocator: *std.mem.Allocator, raw: []const u8, max: usize) ![]std.net.Address {
+    var addresses = try allocator.alloc(std.net.Address, max);
     errdefer allocator.free(addresses);
 
     var index: usize = 0;
     var comma_iterator = std.mem.split(raw, ",");
```

Comment:
```suggestion
    var comma_iterator = std.mem.split(u8, raw, ",");
```

We can land this after the `zig-0.9.0` branch lands on main, so we can already target Zig 0.9.0.

Reply:
:+1: I'll rebase once `zig-0.9.0` is merged.